docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -82,6 +84,8 @@ By default, `XGrid` allows all columns to be resized by dragging the right porti
 To prevent the resizing of a column, set `resizable: false` in the `GridColDef`.
 Alternatively, to disable all columns resize, set the prop `disableColumnResize={true}`.
 
+To restrict resizing a column under a certain width set the `minWidth` property in `GridColDef`.
```

Comment:
Shouldn't we do a maxWidth too?

Reply:
Hmm, that is interesting. It makes sense to add `maxWidth` as well to have all the relevant props supported, but maybe in a separate PR.

Reply:
Created a follow up ticket to not forget about this discussion -> https://github.com/mui-org/material-ui-x/issues/2174